decaf377-fmd/Cargo.toml

```diff
@@ -6,13 +6,18 @@ edition = "2018"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
+penumbra-proto = { path = "../proto/" }
+
+anyhow = "1"
+bytes = "1"
```

Comment:
Hmm, the idea/hope was that the FMD library would be more general than just Penumbra, so having it depend on `penumbra-proto` is less than ideal.  Since the reason to have the dependency is just to be able to implement the `Protobuf` trait, we could avoid it using the same method that we use for `decaf377-rdsa`: make the `penumbra-proto` crate depend on the foreign `decaf377-fmd` crate, and then put the `Protobuf` impls for the `decaf377-fmd` types alongside the `decaf377-rdsa` ones.